dashboard/modules/log/log_head.py

```diff
@@ -78,7 +78,8 @@ async def get_log_from_proxy(self, req) -> aiohttp.web.StreamResponse:
             )
             sr.content_length = r.content_length
             sr.content_type = r.content_type
-            sr.charset = r.charset
+            if r.charset and not sr.content_type.startswith("application/octet-stream"):
```

Comment:
I think we should make it work for state log API as well. (since we will delete this code)

Reply:
sure, i guess let me jsut add a few more test cases for actual binary files.